externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -96,7 +96,16 @@ object SparkSQLEngine extends Logging {
     }
 
     val session = SparkSession.builder().config(sparkConf).enableHiveSupport().getOrCreate()
-    session.sql("SHOW DATABASES")
+    kyuubiConf.get(KyuubiConf.ENGINE_INITIALIZE_SQL).split(";").zipWithIndex
+      .foreach { case (stmt, index) =>
+        val stmtId = s"INITIALIZE_$index"
+        try {
+          session.sparkContext.setJobGroup(stmtId, stmt)
+          debug(session.sql(stmt).queryExecution)
```

Comment:
the thing here is that a query is not eagerly executed. so it needs `.show` or `collect` to trigger